test/runtests.sh

```diff
@@ -1,28 +0,0 @@
-#-------------------------------------------------------------------------------------------------------
```

Comment:
I think you should keep this file, or add it back. Make sure it is set to executable or the CI will not be able to run it.